packages/tests/src/utils/CoreCommandHashing.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity >=0.8.19 <0.9.0;
+
+import { Command } from "../../src/interfaces/IPeripheryProxy.sol";
+
+library CoreCommandHashing {
```

Comment:
do we need to copy paste the entire lib to introduce mock digest calculation?

Reply:
yes, we do (we do in unit tests as well)